lib/commands/fleet/index.ts

```diff
@@ -78,16 +78,28 @@ export default class FleetCmd extends Command {
 			return;
 		}
 
-		const outputApplication = {
-			...application,
+		const applicationToDisplay = {
+			id: application.id,
+			app_name: application.app_name,
+			slug: application.slug,
 			device_type: application.is_for__device_type[0].slug,
 			commit: application.should_be_running__release[0]?.commit,
 		};
 
-		await this.outputData(
-			outputApplication,
-			['app_name', 'id', 'device_type', 'slug', 'commit'],
-			options,
+		if (options.json) {
+			console.log(JSON.stringify(applicationToDisplay, null, 4));
+			return;
+		}
+
+		// Emulate table.vertical title output, but avoid uppercasing and inserting spaces
+		console.log(`== ${applicationToDisplay.app_name}`);
```

Comment:
Is it maybe the time to start using the slug as the header? Since that's what we advertise that ppl should be using :thinking:

Reply:
I don't think so - we advertise people that the slug is the only unique thing but both can be used and are and will (as per previous discussions) still be supported, so I don't think this is really necessary